libs/jacdac/makecodejacdac.ts

```diff
@@ -70,6 +70,12 @@ namespace jacdac {
             configStorage.setItem(DEVICE_NAME_STORAGE_KEY, name);
         };
 
+        jacdac.JACDAC.instance.onIdentificationRequest = function () {
+            // DEVICE_ID_JACDAC_CONFIGURATION_SERVICE = 33
```

Comment:
Couldn't figure out how to refresh dal.d.ts, so I just hacked them in... @pelikhan